content/post/hypriotos-barbossa-for-raspberry-pi-3.md

```diff
@@ -0,0 +1,109 @@
++++
+Categories = ["Docker", "Raspberry Pi", "ARM"]
+Tags = ["Docker", "Raspberry Pi", "ARM"]
+date = "2016-05-16T04:00:00+00:00"
+draft = true
+more_link = "yes"
+title = "HypriotOS 0.8.0 Barbossa for Raspberry Pi 3"
++++
+
+While it is quite hard to make something better that is already good, we feel that we were able to just do that with our latest release of HypriotOS.
+
+__HypriotOS is the easiest and fastest way to get you started with Docker on ARM and IoT devices.__
+
+It is the perfect playground for your first steps with Docker and it allows you to move to advanced stuff like Docker Clustering without breaking a sweat, too.
+
+This is possible because we included a lot of the tools that make the Docker ecosystem awesome - for instance Docker Compose or Docker Swarm - out of the box.
+We also integrated the Hypriot Cluster-Lab into this release, which makes it really easy and painless to set up complex Docker Clusters.
+
+And that's just the tip of the iceberg. So read on to get all the glory details of this new release ... :)
+
+![Raspberry Pi 3](/images/hypriotos-barbossa/iceberg.jpg)
+<div style="text-align:right; font-size: smaller">Image courtesy of [David](https://www.flickr.com/photos/davidw/)</div>
+
+<!--more-->
+
+## Features at a glance
+
+* support for Raspberry Pi Zero, 1, 2 and 3
+* Linux kernel 4.1.19
+* Docker Engine 1.11.0
+* Docker Compose 1.7.1
+* Swarm 1.2.2
+* Cluster-Lab 0.2.6
+* device-init 0.1.5
+
+
+## Updated Docker and Docker-Tools
+Compared to our latest HypriotOS Berry 0.7 we did upgrade the Docker-Engine to v1.11 which now has several binaries instead of one.
+Besides the original Docker binary there is now also a Containerd and a Runc binary. Containerd is basically a supervisor for individual containers that are run by Runc.
+
+The main reasons for splitting up the Docker binary were the efforts to support the [standardization of container technology](https://www.opencontainers.org/) and to make it easier to maintain the existing technology stack.
+You can get a good overview about these changes in the [blog post](https://blog.docker.com/2016/04/docker-engine-1-11-runc/) announcing the release of Docker 1.11.  
+
+And there is also a very intersting deep dive into the details of those changes: 
+
+<iframe id="ytplayer" type="text/html" width="1000" height="560" src="http://www.youtube.com/embed/QL8F2MLCybo" frameborder="0"></iframe>
+
+## Full Raspberry Pi 3 Support
+Noteworthy is also that we now have full support for the Raspberry Pi 3.
+
+With the help of our [flash-tool](https://github.com/hypriot/flash) it is really easy to bring a Raspberry Pi 3 online with only WiFi network connectivity:
+
+```
+flash --ssid MyNetworkName --password SomeSecret https://github.com/hypriot/image-builder-rpi/releases/download/v0.8.0/sd-card-rpi-v0.8.0.img.zip
+```
+
+After that you can usually connect to your Pi by:
+
+```
+ssh pirate@black-pearl
```

Comment:
@StefanScherer That's not necessary for me. I always work without the .local part? Is it necessary for you?